ovos_gui_plugin_shell_companion/brightness.py

```diff
@@ -23,6 +22,8 @@ def __init__(self, bus):
         self.event_scheduler.set_id("ovos-shell")
         self.event_scheduler.set_bus(self.bus)
         self.device_interface = "DSI"
+        self.vcgencmd = shutil.which("vcgencmd")
+        self.ddcutil = shutil.which("ddcutil")
```

Comment:
Not sure if there's any advantage over find executable or which, but PR looks good

Reply:
I have been looking myself over ``stackoverflow`` and the feeling is that starting from Python 3.3, ``shutil`` should be used. If you want I can go back to ``find_executable``